var/spack/repos/builtin/packages/py-scipy/package.py

```diff
@@ -111,6 +111,8 @@ def setup_build_environment(self, env):
         # https://github.com/scipy/scipy/issues/11611
         if self.spec.satisfies('@:1.4 %gcc@10:'):
             env.set('FFLAGS', '-fallow-argument-mismatch')
+            if self.spec.satisfies('^py-numpy@1.16:1.17'):
+                env.set('NPY_DISTUTILS_APPEND_FLAGS', '1')
```

Comment:
This if-statement will never be true since it's inside of another if-statement that contradicts it (1.16–1.17 is not < 1.5). You should reduce the indentation.

Has this PR actually been tested? If you try to install this package, you should see that it doesn't work as is.

Reply:
the spec is:
`py-scipy@:1.4 ^py-numpy@1.16:1.17 ^python@:2 %gcc@10:`
Do you mix the *py-scipy* and *py-numpy* version?

Reply:
Ah yes, you are right. Sorry for the confusion!